modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -91,17 +110,67 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 				gtin = cpInstance.getGtin();
 				height = cpInstance.getHeight();
 				id = cpInstance.getCPInstanceId();
+				incomingQuantityLabel =
+					_cpContentHelper.getIncomingQuantityLabel(
+						cpSkuDTOConverterConvertContext.getCompanyId(),
+						cpSkuDTOConverterConvertContext.getLocale(),
+						cpInstance.getSku(),
+						cpSkuDTOConverterConvertContext.getUser());
 				manufacturerPartNumber = cpInstance.getManufacturerPartNumber();
 				price = _getPrice(
-					cpInstance, 1,
+					cpInstance, cpSkuDTOConverterConvertContext.getQuantity(),
```

Comment:
What if the quantity is not 1 in the quantity selector? How do you manage that?

Reply:
The quantity is a queryParameter, so we can pass it in query after reading it from quantity selector

Reply:
Let's see this together I want to make sure I'm on the same page with you